site/build.sbt

```diff
@@ -62,7 +62,8 @@ lazy val server = (project in file("server"))
       "org.scala-lang" % "scala-compiler" % scalaVersion,
       "org.clapper" %% "classutil" % "1.0.5",
       "com.toddfast.typeconverter" % "typeconverter" % "1.0",
-      "org.scalaz" %% "scalaz-concurrent" % "7.2.0") ++
+      "org.scalaz" %% "scalaz-concurrent" % "7.1.2",
```

Comment:
Maybe we could unify somehow the scalaz dependency version, since we are using different ones along the different modules: 7.1.2, 7.1.4, 7.2.0...